packages/cozy-client/src/models/file.js

```diff
@@ -291,3 +292,268 @@ export const isReferencedByAlbum = file => {
  */
 export const hasMetadataAttribute = ({ file, attribute }) =>
   has(file, `metadata.${attribute}`)
+
+/**
+ * async getFullpath - Gets a file's path
+ *
+ * @param {CozyClient} client - The CozyClient instance
+ * @param {string} dirId  - The id of the parent directory
+ * @param {string} name  - The file's name
```

Comment:
[nit] I would have made this function take two arguments the client, and the file: `getFullPath(client, { dirId, name })`

Reply:
This is relevant, but I think it's better to keep the current function to ease migration

Reply:
nit: this method should also be renamed to fetchFullPatch. But the work done in this PR is already 👍 so let's keep that in mind for the future